src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy

```diff
@@ -251,13 +251,21 @@ class ProtobufPlugin implements Plugin<Project> {
       if (variant.hasProperty("compileConfiguration")) {
         // For Android Gradle plugin >= 2.5
         Attribute artifactType = Attribute.of("artifactType", String)
-        Task extractIncludeProtosTask =
-            maybeAddExtractIncludeProtosTask(
-                    variant.name,
-                    variant.compileConfiguration.incoming.artifactView{ attributes{ it.attribute(artifactType, "jar") }}.files,
-                    variant.hasProperty("testedVariant") ?
-                            variant.testedVariant.compileConfiguration.incoming.artifactView{ attributes{ it.attribute(artifactType, "jar") }}.files :
-                            null)
+        String name = variant.name
```

Comment:
My intellij was complaining that the types could not be inferred, and explicitly assigning the vars this way made it quiet